ParkenDD/LotlistView/LotlistViewController.swift

```diff
@@ -17,18 +17,18 @@ class LotlistViewController: UITableViewController, UIViewControllerPreviewingDe
 
 	var dataURL: String?
 
-    var dataSource = LotlistDataSource()
+	var dataSource = LotlistDataSource()
```

Comment:
Not only are you changing the indentation, but doing so adds to the complexity of the code review. Now I need to check every green/red line to see if it's a legit change or an indentation change.

Add a commit and fix it.